src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
The constructor of Circle should not generate or manipulate the radius value directly. Instead, consider passing a pre-generated value to the constructor for better separation of concerns and readability.